plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -185,6 +185,12 @@ func (e *Elasticsearch) Init() error {
 	return nil
 }
 
+// This is here to allow us to conform to the ServiceInput interface, which is
+// required for Stop() to be called.
+func (e *Elasticsearch) Start(_ telegraf.Accumulator) error {
```

Comment:
We end up converting a bunch of plugins into `telegraf.ServiceInput` plugins, because we need the `Stop()` hook to work. It's not very pretty :grimacing:

Reply:
Please remove the comment, it's nothing wrong with implementing this as-is and putting a dummy method there to fulfill the interface requirements!